source/assets/stylesheets/styles/_news_tile.scss

```diff
@@ -3,15 +3,19 @@
 }
 
 .news-tile__container {
- @extend .blog-medium-tile__container;
+  @extend .blog-medium-tile__container;
+
+  @include breakpoint($tablet-breakpoint) {
+    height: 386px;
```

Comment:
I suggest changing the `.blog-medium-tile__container` min-height to `388px` instead of having this override as the problem may occur in blogs too but more so to prevent overlapping margins which this fixed height introduces:

with `height` (notice the 2px margin overflow)
<img width="389" alt="screen shot 2016-03-04 at 12 34 39" src="https://cloud.githubusercontent.com/assets/885223/13527560/b8c3871c-e207-11e5-87ee-4463df6a0555.png">

with `min-height` adjusted
<img width="446" alt="screen shot 2016-03-04 at 12 35 49" src="https://cloud.githubusercontent.com/assets/885223/13527561/b8c40962-e207-11e5-9cc2-895a8510c91c.png">